-------------------------------------------------------------------------------
Test set: chainOfResponsibility.ServicioTecnicoTest
-------------------------------------------------------------------------------
Tests run: 4, Failures: 3, Errors: 0, Skipped: 0, Time elapsed: 0.051 s <<< FAILURE! -- in chainOfResponsibility.ServicioTecnicoTest
chainOfResponsibility.ServicioTecnicoTest.creacionServicioTecnicoIdFallidaTest -- Time elapsed: 0.006 s <<< FAILURE!
org.opentest4j.AssertionFailedError: Expected java.lang.IllegalArgumentException to be thrown, but nothing was thrown.
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:152)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:73)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:35)
	at org.junit.jupiter.api.Assertions.assertThrows(Assertions.java:3115)
	at chainOfResponsibility.ServicioTecnicoTest.creacionServicioTecnicoIdFallidaTest(ServicioTecnicoTest.java:21)
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)

chainOfResponsibility.ServicioTecnicoTest.creacionServicioTecnicoNombreFallidaTest -- Time elapsed: 0.002 s <<< FAILURE!
org.opentest4j.AssertionFailedError: Expected java.lang.IllegalArgumentException to be thrown, but nothing was thrown.
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:152)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:73)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:35)
	at org.junit.jupiter.api.Assertions.assertThrows(Assertions.java:3115)
	at chainOfResponsibility.ServicioTecnicoTest.creacionServicioTecnicoNombreFallidaTest(ServicioTecnicoTest.java:27)
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)

chainOfResponsibility.ServicioTecnicoTest.creacionServicioTecnicoTelefonoFallidaTest -- Time elapsed: 0.001 s <<< FAILURE!
org.opentest4j.AssertionFailedError: Expected java.lang.IllegalArgumentException to be thrown, but nothing was thrown.
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:152)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:73)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:35)
	at org.junit.jupiter.api.Assertions.assertThrows(Assertions.java:3115)
	at chainOfResponsibility.ServicioTecnicoTest.creacionServicioTecnicoTelefonoFallidaTest(ServicioTecnicoTest.java:33)
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)

